arangod/Utils/CollectionNameResolver.cpp

```diff
@@ -32,7 +32,33 @@
 #include "VocBase/LogicalView.h"
 #include "VocBase/vocbase.h"
 
-using namespace arangodb;
+namespace arangodb {
+
+std::shared_ptr<LogicalCollection> CollectionNameResolver::getCollection(
+  TRI_voc_cid_t id
+) const noexcept {
+  #ifdef ARANGODB_ENABLE_MAINTAINER_MODE
+    return std::dynamic_pointer_cast<LogicalCollection>(getDataSource(id));
```

Comment:
please add TRI_ASSERT:
```
auto col = std::dynamic_pointer_cast<LogicalCollection>(getDataSource(id))
TRI_ASSERT(col)
return col
```

Reply:
no need for TRI_ASSERT, it is valid to return nullptr